staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -35,16 +35,25 @@ var panicOnUnknown = false
 
 // builtInFunctions is a list of functions used in cost tests that are not handled by CostEstimator.
 var knownUnhandledFunctions = map[string]bool{
-	"uint":          true,
-	"duration":      true,
-	"bytes":         true,
-	"timestamp":     true,
-	"value":         true,
-	"_==_":          true,
-	"_&&_":          true,
-	"_>_":           true,
-	"!_":            true,
-	"strings.quote": true,
+	"@not_strictly_false": true,
```

Comment:
curious what in our test triggers this

Reply:
The conditions of the new comprehensions.  xref: https://github.com/google/cel-go/blob/8ad600b649be1b9ef5a003e8c5632d89b9aaf790/ext/comprehensions.go#L236